### PR TITLE
TERM_STATUS_BYTE_MASK definition

### PR DESCRIPTION
**Pull request by bogo8liuk**

### CODE REVIEW

**Review by daddinuz — APPROVED** [review]

File: include/term.h [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 #include <primitive_types.h>
 
+#define TERM_STATUS_BYTE_MASK 0xFFU
```
Comment (Segandrea) [inline include/term.h]: Change this macro into a function

File: sources/handlers.c [hunk 2]
```diff
@@ -78,7 +78,7 @@ void handlers_interruptHandler(void) {
             }
 
             // TODO: extract bitmask in a separate module, look term module
```
Comment (Segandrea) [inline sources/handlers.c]: Remove TODO comments

File: include/term.h [hunk 3]
```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <primitive_types.h>
+#include <machine.h>
 
 #define TERM_STATUS_BYTE_MASK 0xFFU
```
Comment (Segandrea) [inline include/term.h]: Remove this macro

File: include/term.h [hunk 4]
```diff
@@ -69,3 +70,23 @@ extern bool term_getchar(char *buf);
...
  */
 extern usize term_gets(bool (*p)(char), char *buf, usize n);
+
+/**
+ * Gets the status of transmission
+ *
+ * @attention if tp is NULL, it causes kernel PANIC
+ *
+ * @param tp The terminal
+ * @return Bit-wise And between terminal transmit status and the byte mask
```
Comment (Segandrea) [inline include/term.h]: It's not necessary to expose the fact that the status is given by a bitwise and

**Review by Segandrea — CHANGES_REQUESTED** [review]

**Review by Segandrea — APPROVED** [review]

**Pull request merged**